proposals/templates/proposals/proposal_diff.html

```diff
@@ -209,7 +210,7 @@ <h2 class="mt-5 mb-1">{% trans "Eén of meerdere trajecten?" %}</h2>
                 {% endif %}
             </table>
 
-            {% if proposal.wmo.status == proposal.wmo.NO_WMO %}
+            {% if proposal.wmo.status == proposal.wmo.NO_WMO or proposal.wmo.status == proposal.wmo.JUDGED %}
             {% for study, p_study in zipped_studies %}
```

Comment:
I like the filter solution more than the context var, could we use it here as well?

Reply:
Yes we can, and now we do!

Good spot, missed it yesterday.